google/cloud/firestore_bundle/services/__init__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
it seems the generated code is in this PR along with some manual edits. Would it be possible to separate the gen portions to shrink the size a bit?

Reply:
That's a great idea.